### DISCUSSION TITLE
Why is #1176 closed?

### DISCUSSION THREAD

**Discussion by QZAiXH** [slackhq/nebula#1209] [General]
I am very excited about the potential of IPv6, so when I noticed the relevant submission in the PR, I felt a surge of enthusiasm. However, upon checking today, I found that the PR has been closed. Could you please explain the reason for this?

**Comment by IanVS** [comment]
Don't worry, IPv6 is still being actively developed.  It's a large feature, so there may be some intermediate PRs that are opened and then either closed or merged, depending on the direction taken.
